tests/ident/test_engine_parser_omssa.py

```diff
@@ -189,16 +195,19 @@ def test_engine_parsers_omssa_next():
             "Raw data location": "/Users/cellzome/Dev/Gits/Ursgal/ursgal_master/example_data/test_Creinhardtii_QE_pH11.mgf",
         },
     )
-    row = next(parser)
-    assert isinstance(row, UnifiedRow)
-    print(row.data.keys())
+    df = parser.get_dataframe()
+    row = df.iloc[0, :]
     assert row["Sequence"] == "ALAMEWGPFPRLMVVACNDAINVCRK"
-    assert row["Modifications"] == "Oxidation:4;Carbamidomethyl:17;Carbamidomethyl:24"
+    assert row["Modifications"] == "Carbamidomethyl:17;Carbamidomethyl:24;Oxidation:4"
```

Comment:
Modifications have to be sorted by position!

Reply:
sorted by position and by mod name secondly in 9fa4b5a